integrations/client/test_delphi_epidata.py

```diff
@@ -283,15 +283,14 @@ def test_covidcast_meta(self):
     # 1st issue: 0 10 20
     # 2nd issue: 1 11 21
     # 3rd issue: 2 12 22
-    rows = [
-      CovidcastTestRow.make_default_row(
-        time_value=DEFAULT_TIME_VALUE + t,
-        issue=DEFAULT_ISSUE + i,
-        value=t*10 + i
-      )
-      for i in range(3) for t in range(3)
-    ]
-    self._insert_rows(rows)
+    for i in range(3):
+      for t in range(3):
+        row = CovidcastTestRow.make_default_row(
+          time_value=DEFAULT_TIME_VALUE + t,
+          issue=DEFAULT_ISSUE + i,
+          value=t*10 + i
+        )
+        self._insert_rows([row])
```

Comment:
the test is failing because the data points all get re-issued, so the minimum issue across all of the "latest" data is actually that most recent issue.  we can add another (unique) day that doesnt have its issue updated.

i think this should work:
```suggestion
        self._insert_rows([row])
    self._insert_rows([CovidCastTestRow.make_default_row(
      time_value=DEFAULT_TIME_VALUE-1,
      issue=DEFAULT_ISSUE,
      value=12
    )])
```
and then youll have to change below to include
```py
      min_time=DEFAULT_TIME_VALUE-1,
```
and probably a new `stdev_value` too

Reply:
Thanks for the various fixes. I've been struggling with this test